include/ptracer.h

```diff
@@ -1,8 +1,8 @@
-/* 
+/*
  * Copyright (c) 2024, The Pallene Developers
  * Pallene Tracer is licensed under the MIT license.
```

Comment:
It's still an STB header. I would like to keep it in a separate `lib/` directory, because the header also includes non-inlined code. It's neither a proper header file (technically by extension it is, but it's not serving only the symbol resolving purpose) nor a proper source file.

It serves a proper Header Static Library.

Reply:
I  added another commit. All files now live in the root and there there are no  "lib" and "src" directories. I changed all the `include <ptracer.h>` into `include "ptracer.h"`, so we no longer need a "-I" compiler flag.

Note: https://github.com/nothings/stb also puts everything in the root instead of in a lib directory.